lib/fuse_lowlevel.c

```diff
@@ -2907,6 +2907,28 @@ static void fuse_ll_pipe_destructor(void *data)
 	fuse_ll_pipe_free(llp);
 }
 
+static unsigned int get_max_pages(void)
+{
+	char buf[32];
+	int res;
+	int fd;
+
+	fd = open("/proc/sys/fs/fuse/max_pages_limit", O_RDONLY);
+	if (fd < 0)
+		return FUSE_MAX_MAX_PAGES;
+
+	res = read(fd, buf, sizeof(buf) - 1);
+
+	close(fd);
+
+	if (res < 0)
+		return FUSE_MAX_MAX_PAGES;
```

Comment:
While you are there, would you have time to change this name? I think your kernel patch changed to FUSE_MAX_PAGES_LIMIT?

Reply:
Will do!